### ISSUE TITLE
Set seed

### ISSUE THREAD

**Issue by ariG23498** [AakashKumarNain/TF_JAX_tutorials#3]
A neat way to setting seed is by using [`tf.keras.utils.set_random_seed(SEED)`](https://www.tensorflow.org/api_docs/python/tf/keras/utils/set_random_seed). This API basically does away with all the redundant `numpy`, `random`, and `tf random` seed setting code.

Hope this will find its way into the tutorials. Thanks for the tutorial BTW.

**Comment by AakashKumarNain** [comment]
Thanks @ariG23498 I will include this in the revised version
